consistent-sampling/src/main/java/io/opentelemetry/contrib/sampler/consistent56/ConsistentRateLimitingSampler.java

```diff
@@ -120,36 +157,88 @@ public State(double effectiveWindowCount, double effectiveWindowNanos, long last
     this.inverseAdaptationTimeNanos = NANOS_IN_SECONDS / adaptationTimeSeconds;
     this.targetSpansPerNanosecondLimit = NANOS_IN_SECONDS * targetSpansPerSecondLimit;
 
-    this.state = new AtomicReference<>(new State(0, 0, nanoTimeSupplier.getAsLong()));
+    double t = 1.0 / (targetSpansPerSecondLimit * adaptationTimeSeconds);
+    this.probabilitySmoothingFactor = t / (1.0 + t);
```

Comment:
could you please comment on the rationale behind this choice of the smoothing factor?

Reply:
Sure, added comments.